libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/gutenberg/GutenbergEditorFragment.java

```diff
@@ -457,6 +473,27 @@ public void onActivityResult(int requestCode, int resultCode, @Nullable Intent d
         return otherMediaOptions;
     }
 
+    private ArrayList<MediaOption> initOtherMediaFileOptions() {
+        ArrayList<MediaOption> otherMediaOptions = new ArrayList<>();
+
+        Bundle arguments = getArguments();
+        FragmentActivity activity = getActivity();
+        if (activity == null || arguments == null) {
+            AppLog.e(T.EDITOR,
+                    "Failed to initialize other media options because the activity or getArguments() is null");
+            return otherMediaOptions;
+        }
```

Comment:
We aren't using `arguments` here so we can remove the variable, the condition from the if statement about arguments, and modify the message being logged as well not to include `getArguments()`

Reply:
Fixed.